src/main/webapp/dev/js/common/instructorFeedbackResults.js

```diff
@@ -149,11 +149,41 @@ function replaceButtonHtmlAndTooltipText(button, from, to) {
 function expandOrCollapsePanels(expandCollapseButton, panels) {
     const STRING_EXPAND = 'Expand';
     const STRING_COLLAPSE = 'Collapse';
+    const expandCollapseStudentsButton = 'a[id^="collapse-panels-button-section-"]';
+    const expandCollapseTeamButton = 'a[id^="collapse-panels-button-team-"]';
+    const isButtonInExpandMode = $(expandCollapseButton).html().trim().startsWith(STRING_EXPAND);
+
+    /*
+     *When collapsing expandCollapseButton, all panels buttons under it,
+     * (expandCollapseTeamButton and  expandCollapseStudentButton) should be in collapsed state.
+     * So that next time we expand the main panel, we find the expandCollapseTeamButton and
+     * expandCollapseStudentButton in collapsed state.
+     */
+    if (!panels && !isButtonInExpandMode) {
+        if ($(expandCollapseStudentsButton).length
+            && $(expandCollapseStudentsButton).html().trim().startsWith(STRING_EXPAND)) {
+            $(expandCollapseStudentsButton).trigger('click');
+            $(expandCollapseStudentsButton).trigger('mouseleave');
```

Comment:
I just realised a better approach: since at the end of the operation the panels themselves are actually correctly collapsed and only the button label and tooltip are wrong, we just need to use `replaceButtonHtmlAndTooltipText` on the buttons, right?

Reply:
Yes, using `replaceButtonHtmlAndTooltipText` is a better approach. :+1: 
Triggered `mouseleave` to hide the tooltip that showed up after using above mentioned.